.github/workflows/pythonpackage.yml

```diff
@@ -0,0 +1,38 @@
+name: CI Tests
+
+on: [pull_request]
+
+jobs:
+  build:
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        os: [ubuntu-latest, macos-latest, windows-latest]
+        python-version: ["3.8", "3.9", "3.10", "3.11"]
+    defaults:
+      run:
+        shell: bash
+    steps:
+      - uses: actions/checkout@v3
+      - name: Set up Python ${{ matrix.python-version }}
+        uses: actions/setup-python@v4
+        with:
+          python-version: ${{ matrix.python-version }}
+          architecture: x64
+      - name: Install dependencies
+        run: |
+          python -m pip install --upgrade pip
```

Comment:
Does this work without installing flit?

python -m pip install --upgrade pip flit

Reply:
Oh wow, it does!

It works on the CI tests and just tested on fresh venv and `python -m pip install .` seems to read the toml file install in the same way as flit. It even exposes the cli cmd.

Reply:
>Oh wow, it does!
>It works on the CI tests and just tested on fresh venv and python -m pip install . seems to read the toml file install in the same way as flit. It even exposes the cli cmd.

Ah, my bad :) Flit is only specified as the build backend in pyproject.toml and python3 -m pip install . works fine. So everything is as it should be.

I will update the readme to also use 

python3 -m pip install . 

instead of 

flit install 

after this merge. It is currently not wrong but let's keep it consistent everywhere.